build: remove build breaking unicode filename

nbrugger-tgm opened the pull request:
The file contains non-standard (UTF-8, ASCII) but Unicode characters (in the file name) which bricks the build on my machine with "FileNotFoundException: ??????-0.0.yml"

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/micronaut-projects/micronaut-core?pullRequest=9001) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/micronaut-projects/micronaut-core?pullRequest=9001) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/micronaut-projects/micronaut-core?pullRequest=9001) it.</sub>

nbrugger-tgm committed:
build: remove build breaking unicode filename

    The file contains non-standard (UTF-8, ASCII) but unicode characters (in the file name) which brick the build on my machine with "FileNotFound: ??????-0.0.yml"

nbrugger-tgm commented:
There is a test that verifies the existence of the file (or rather is correct serving by netty).
Is it maybe possible to remove the file from gradle indexing - any gradle command fails for me when the file is there

```
Execution failed for task ':http-server-netty:processTestResources'.
> Cannot access input property 'rootSpec$1' of task ':http-server-netty:processTestResources'. Accessing unreadable inputs or outputs is not supported. Declare the task as untracked by using Task.doNotTrackState(). See https://docs.gradle.org/8.0.2/userguide/incremental_build.html#disable-state-tracking for more details.
   > Failed to create MD5 hash for file '~/micronaut-core/http-server-netty/src/test/resources/public/??????-0.0.yml' as it does not exist.
```

graemerocher commented:
you can probably skip the task for now `./gradlew test -X http-server-netty:processTestResources`

nbrugger-tgm commented:
So this pr is dismissed/to close then?
It might be a problem more people run into (maybe) but it seems OS specific or on other environmental factors.

For me, it would be ok to close it if you don't see it as a big issue or no good solution is available.

nbrugger-tgm commented:
I still think this should be fixed but this PR is too old and it seems to be unwanted
